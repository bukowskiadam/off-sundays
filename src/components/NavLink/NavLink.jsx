import React from 'react';
import Link from 'gatsby-link';

import style from './NavLink.module.css';

const NavLink = props => (
    <p className={style.link}>
        <Link {...props} />
    </p>
);

export default NavLink;
