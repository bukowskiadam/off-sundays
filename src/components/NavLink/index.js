import React from 'react';
import Link from 'gatsby-link';

import style from './index.module.css';

export default props => (
    <p className={style.link}>
        <Link {...props} />
    </p>
);
