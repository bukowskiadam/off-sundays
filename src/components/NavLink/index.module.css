.link a {
    color: #005a9d;
}
