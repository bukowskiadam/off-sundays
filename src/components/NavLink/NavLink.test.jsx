import React from 'react';
import { shallow } from 'enzyme';

import NavLink from './NavLink';

describe('NavLink', () => {
    it('renders correctly', () => {
        const wrapper = shallow(
            <NavLink to="nowhere" />,
        );

        expect(wrapper).toMatchSnapshot();
    });
});
