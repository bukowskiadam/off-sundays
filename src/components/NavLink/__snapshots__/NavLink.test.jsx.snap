// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavLink renders correctly 1`] = `
<p
  className="link"
>
  <GatsbyLink
    to="nowhere"
  />
</p>
`;
